dbt/include/fabric/macros/materializations/models/view/view.sql

```diff
@@ -20,7 +12,7 @@
 
   {% if (existing_relation != none) %}
     -- drop the temp relations if they exist already in the database
-    {{ drop_relation_if_exists(backup_relation) }}
+    {{ drop_relation(backup_relation) }}
```

Comment:
@prdpsvs To fix the failing caching tests, I think you need to change this to:
- `drop_relation_if_exists`
- `adapter.drop_relation`

https://github.com/dbt-labs/dbt-adapters/issues/221